src/platform-includes/session-replay/install/javascript.mdx

```diff
@@ -17,7 +17,7 @@ entire Sentry SDK. You have to add it in addition to the Sentry Browser SDK bund
 -->
 
 <script
-  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
+  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.tracing.min.js"
```

Comment:
Not sure but I think we already changed this once to the browser+tracing bundle. I'd vote to show this bundle as the older one didn't include performance monitoring functionality.